client/me/help/help-contact-form/index.jsx

```diff
@@ -28,12 +28,22 @@ import { selectSiteId } from 'state/help/actions';
 import { getHelpSelectedSite } from 'state/help/selectors';
 import wpcomLib from 'lib/wp';
 import HelpResults from 'me/help/help-results';
+import {
+	bumpStat,
+	recordTracksEvent,
+	composeAnalytics,
+} from 'state/analytics/actions';
 
 /**
  * Module variables
  */
 const wpcom = wpcomLib.undocumented();
 
+const trackSibylClick = ( event, helpLink ) => composeAnalytics(
+	bumpStat( 'sibyl_question_clicks', helpLink.id ),
+	recordTracksEvent( 'calypso_sibyl_question_click', {} )
```

Comment:
You can omit the second arg if we don't have anything extra to pass. Would the helplink id be helpful here?